<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Breno Livio Silva de Almeida</title>
  <link rel="stylesheet" href="style.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.0/css/all.min.css">
</head>
<body>
  <header>
    <nav>
        <div class="main-links">
            <a href="/">Home</a>
            <a href="/publications">Publications</a>
            <a href="/cv/Breno_CV.pdf">CV</a>
        </div>

        <div class="social-links">
            <a href="https://scholar.google.com/citations?user=g0zZ_U4AAAAJ" title="Google Scholar" target="_blank"><i class="fas fa-graduation-cap"></i></a>
            <a href="https://www.linkedin.com/in/brenoslivio" title="LinkedIn" target="_blank"><i class="fab fa-linkedin"></i></a>
            <a href="https://github.com/brenoslivio" title="GitHub" target="_blank"><i class="fab fa-github"></i></a>
            <a href="mailto:brenoslivio@pm.me" title="Email"><i class="fas fa-envelope"></i></a>
        </div>
    </nav>
  </header>

  <div class="container">
    <aside>
      <h3>ABOUT ME</h3>
      <ul>
        <li><a href="/cv/Breno_CV.pdf">CV</a></li>
        <li><a href="#">Publications</a></li>
      </ul>

      <!-- <h3>SOFTWARE</h3>
      <ul>
        <li><a href="#">blabla</a></li>
        <li><a href="#">blabla</a></li>
      </ul> -->

      <h3>OTHER STUFF</h3>
      <ul>
        <li><a href="#">FAQ</a></li>
      </ul>
    </aside>

    <main>
      <section>
        <h1>Who Am I?</h1>
        <p>I am a double-degree PhD student at the <a href="https://www.icmc.usp.br/">
          Institute of Mathematics and Computer Sciences</a> (ICMC) of the 
          <a href="https://www5.usp.br/">University of São Paulo</a> (USP) and the 
          <a href="https://www.uni-leipzig.de/en">University of Leipzig</a>, researching 
          pattern recognition in biological sequences using alignment-free methods such 
          as language models. My work focuses on developing computational approaches to 
          analyze complex biological data, bridging machine learning and bioinformatics.
        </p> 

        <p>As part of my doctoral research, I am also a Research Assistant at the 
          <a href="https://www.ufz.de/">Helmholtz Centre for Environmental Research (UFZ)</a> 
          in the Microbial Data Science Group headed by <a href="https://scholar.google.com/citations?user=BOIkqjQAAAAJ">
          Dr. Ulisses da Rocha</a> (Department of Computational Biology and Chemistry). 
          Here, I apply and extend my PhD methods to study microbial systems, 
          contributing to large-scale environmental data analysis 
          and advancing tools for sequence-based discovery.
        </p> 
        
        <p>Before my PhD, I completed my BSc in Computer Science at ICMC-USP, where I first 
          explored machine-learning applications in bioinformatics. During this time, I had 
          the privilege of working under the guidance of 
          <a href="https://scholar.google.com/citations?user=Jx_5GrgAAAAJ">Prof. Dr. André de Carvalho</a>, 
          who played a pivotal role in shaping my research interests. Now, as a PhD student, 
          I am fortunate to continue this academic journey with him as one of my co-advisors, 
          alongside <a href="https://scholar.google.com/citations?user=pVnGRlkAAAAJ">Prof. Dr. Peter Stadler</a> 
          from the University of Leipzig.
        </p>
      </section>

      <section>
        <h1>Research Interests</h1>
        <p>
          Since High School, where I developed a simple Chatbot for assistance in interventions
          with patients with Asperger's Syndrome, I tried to use computer science for social good.
          I continued doing this in college, where my interests in biology and nutrition led me to
          try to understand disorders such as Autism Spectrum and Bipolar Disorder through the lens of
          machine learning and bioinformatics.
        </p>

        <p>
          Since 2021, I have been continuing to work on computer science for social good by employing
          methods for democratizing machine learning, such as using automated machine learning (AutoML),
          which can help non-experts to use machine learning.
        </p>

        <p>
          My current research interests, in a more <b>technical version</b>: 
        </p>

        <p>
          I am interested in developing alignment-free computational approaches, particularly machine 
          learning-based methods to analyze biological sequences that resist traditional annotation 
          due to high divergence and remote homology. By leveraging language models to extract evolutionary 
          and ecological features, I aim to build nature-inspired machine-learning frameworks for the 
          discovery of novel enzymes.
        </p>

        <p>
          Now, in a <b>less technical version that I could explain to my mom and impress her</b>: 
        </p>

        <p>
          Proteins are molecules that perform many essential functions in the body, 
          such as building tissues and regulating processes. Some researchers like to call proteins 
          with unknown functions as being "functional dark matter"
          <a href="https://merenlab.org/2017/06/22/microbial-dark-matter/">(although some are pretty much 
          against using the term "dark matter")</a>.
        </p>

        <p>
          Even with databases containing billions of proteins, those proteins are not considered similar
          to those with functions in the databases, so we cannot adequately characterize them. The problem
          with the methods that do this comparison is that they can be prone to errors. 
        </p>

        <p>
          These methods compare protein sequences to find similarities, but they can miss highly 
          divergent sequences, which are proteins that have changed so much over time that their shared origins are hard 
          to detect. They also struggle with remote homologs: distantly related proteins with the 
          same evolutionary ancestor but very different sequences. Because these methods rely on apparent 
          similarities, many important but deeply evolved proteins remain unknown, leaving gaps in our 
          understanding of "functional dark matter."
        </p>

        <p>
          Just as language models, like the one used by ChatGPT, can learn patterns in words in a sentence, they can also
          learn patterns in amino acids of proteins, finding hidden patterns in the "language of life" such evolutionary
          and ecological information. I believe language models can be key to unlocking nature's hidden biochemical 
          potential by finding novel enzymes, discovering new enzymes that could revolutionize medicine, industry, 
          and sustainability.
        </p>
        
      </section>

      <section>
        <h1>TBD</h1>
        <p><strong>July 2025:</strong> blabla</p>
      </section>


    </main>

    <figure>
      <img src="imgs/me_square.png" id="profileImage" alt="Profile photo" class="profile-pic">
      <figcaption id="profileCaption">That's me! (try clicking on it)</figcaption>
    </figure>
    
  </div>
  <script src="js/script.js"></script>
</body>
</html>
